src/util/eventHandlers.tsx

```diff
@@ -3,58 +3,108 @@ import * as React from "react";
 import { getCaretCoordinates } from "./index";
 
 export function getHandlers({
-  ref,
-  suggestions,
-  setShowSuggestion,
-  showSuggestion,
-  setFocusIndex,
-  focusIndex,
-  setCaret
-}) {
+                              ref,
```

Comment:
not quite sure am I using the different format tooling

Reply:
so it turns out that you have to manully set Webstorm to use project .prettier config file